src/Streamly/Internal/Data/Unfold/Enumeration.hs

```diff
@@ -142,12 +144,16 @@ enumerateFromThenNum =
 
 -- | Same as 'enumerateFromStepNum' using a stride of 1:
 --
--- >>> enumerateFromNum = lmap (\from -> (from, 1)) enumerateFromStepNum
+-- @
+-- > enumerateFromNum = lmap (\from -> (from, 1)) enumerateFromStepNum
+-- @
 --
 -- Also, same as 'enumerateFromThenNum' using a stride of 1 but see the note in
 -- 'enumerateFromThenNum' about the loss of precision:
 --
--- >>> enumerateFromNum = lmap (\from -> (from, from + 1)) enumerateFromThenNum
+-- @
+-- > enumerateFromNum = lmap (\from -> (from, from + 1)) enumerateFromThenNum
+-- @
```

Comment:
By disabling docspec on these we will not be able to check these for correctness. I do not see any reason why docspec should not work on these.